vertical-pod-autoscaler/README.md

```diff
@@ -214,6 +218,18 @@ kubectl delete clusterrolebinding myname-cluster-admin-binding
 
 # Known limitations
 
+## Limits control
+
+When setting limits VPA will conform to
+[resource policies](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2/types.go#L82).
+It will maintain limit to request ratio specified for all containers.
+
+VPA will also attempt to fit within
+[limit ranges](https://kubernetes.io/docs/concepts/policy/limit-range/).
+However if maintaining limit/request ratio, following resource policies, and
```

Comment:
This sentence is very difficult to parse. I think we should write that we try to cap to max/min limit, but that VPA policy (minRecommended/maxRecommended) take precedence. It may clarify things a bit if we give concrete examples maybe?

Reply:
Concrete examples = concrete request,limit,ratio, limitrange values?

Reply:
Reworded and added examples